source/redirect-latest.html.slim

```diff
@@ -0,0 +1,5 @@
+- content_for(:senarios) do
```

Comment:
No worries :).  Mind fixing it?  Seems necessary for it to work properly.  I'll be happy to merge once that's fixed.